config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml

```diff
@@ -3041,6 +3041,7 @@ presubmits:
             memory: 6Gi
         securityContext:
           privileged: true
+
```

Comment:
sorry, it shows just adding a newline here. It was intended to remove something, correct?

Reply:
yeah, I made a mistake on rebasing

Reply:
fixed now